utils/litemap/tests/store.rs

```diff
@@ -113,6 +120,11 @@ impl<'a, K: 'a, V: 'a> StoreIterableMut<'a, K, V> for VecWithDefaults<(K, V)> {
         IntoIterator::into_iter(self.0)
     }
 
+    #[inline]
```

Comment:
now revert here.

Reply:
There is a similar impl for vec https://github.com/unicode-org/icu4x/blob/main/utils/litemap/src/store/vec_impl.rs#L124-L127

Reply:
Reverted this impl.